### DISCUSSION TITLE
Redirect issue with Joomla sites

### DISCUSSION THREAD

**Discussion by tassosm** [laravel/valet#1139] [Q&A]
I am running Valet 2.15.0 on macOS Big Sur (M1) and I am facing the following strange issue with Joomla sites.

At random times when I open any URL under the /administrator/ directory, I get redirected to a URL version without "/administrator/". For instance, the URL https://dev.test/administrator/index.php?option=com_contact redirects me to https://dev.test/index.php?option=com_contact which is an invalid URL. 

## Steps to Reproduce
Unfortunately, there are no exact steps to follow to reproduce this issue as it happens randomly. And when it happens, `valet restart` is enough to fix the issue.

## Diagnosis
Running diagnostics... (this may take a while)
24/40 [================>-----------]  60%
In Process.php line 436:
                                                  
The process has been signaled with signal "6".

**Comment by stergosz** [comment]
I can confirm this happens on my end as well, at random times.
